src/io/flutter/logging/FlutterLogView.java

```diff
@@ -39,8 +37,32 @@
 import java.awt.datatransfer.StringSelection;
 import java.awt.event.MouseEvent;
 
+import static com.intellij.ui.SimpleTextAttributes.STYLE_PLAIN;
+
 public class FlutterLogView extends JPanel implements ConsoleView, DataProvider, FlutterLog.Listener {
 
+  private class EntryModel implements FlutterLogTree.EntryModel {
+    boolean color;
```

Comment:
we should rename that variable. Eg, `isEnableLogColor` or something like that.
(`color` feels like the actual color value - Colors.RED)

Reply:
In a private classes I prefer shorter names, especially when the meaning should be clear from the context (and the type).  Of course, the fact that it surprised you, means it wasn't clear! 😄  I'll make it better.

Thanks!